cypress/integration/1-getting-started/Local tests/one/2_Search_bar_functionality.spec.js

```diff
@@ -21,20 +21,25 @@ describe("Test search bar functionality", () => {
     it("should be able to test the search bar functionality", () => {
       cy.get(".modal-header>:last-child()").click(); // Here we close the start tour pop up
       cy.get('[data-test-id="input-search"]').type("admin"); // We search for the Admin app
-      cy.wait(1000); // With this command we wait 1s before the execution of the next command. We use this for better visibility
-      cy.get('[data-test-id="input-search"]').clear().type("CRM"); // We search for the CRM app
-      cy.wait(1000);
+      cy.get('[data-test-id="Admin Area"]').click({ force: true });
```

Comment:
I would suggest splitting this test into smaller ones. Create a context() with it() containing a test for each webapp. This would make things a lot simpler and better to read/debug